sunpy/net/hek2vso/tests/test_hek2vso.py

```diff
@@ -49,14 +50,14 @@ def vso_client():
 def test_translate_results_to_query(hek_client):
     """Make sure that conversion of HEK results to VSO queries is accurate"""
     h = hek_client
-    hek_query = h.search(hekTime, hekEvent)
+    hek_query = h.search(hekTime, hekEvent, a.hek.FL.PeakFlux > 1000)
```

Comment:
Why the extra bit of the query?

Reply:
To reduce the number of records and therefore speed up the test.